### PR TITLE
ci: enable Android 12 for tests on CI

### PR DESCRIPTION
**Pull request by AndrewKushnir**
This commit updates SauceLabs config to use Android 12 and drop Android 10, so we maintain 2 most recent versions according to https://angular.io/guide/browser-support.

## PR Type
What kind of change does this PR introduce?

- [x] CI related changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: browser-providers.conf.js [hunk 1]
```diff
@@ -19,22 +19,22 @@ const config = {
 const debugMode = false;
 
 const customLaunchers = {
-  'SL_ANDROID10': {
```
Comment (devversion) [inline browser-providers.conf.js:22]: You need to update the `config` keys above here to reflect the new browsers.

File: browser-providers.conf.js [hunk 2]
```diff
@@ -45,8 +45,8 @@ const sauceAliases = {
 };
 
 module.exports = {
-  customLaunchers: customLaunchers,
-  sauceAliases: sauceAliases,
+  customLaunchers : customLaunchers,
+  sauceAliases : sauceAliases,
```
Comment (devversion) [inline browser-providers.conf.js:49]: Super nit: Not sure where this spacing is coming from, and why clang doesn't complain here.

**Review by devversion — DISMISSED** [review]

File: browser-providers.conf.js (hunk 1 above)
Comment (AndrewKushnir) [reply browser-providers.conf.js:22]: Thanks Paul! 👍

**Review by devversion — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
Merge-assistance: the `aio_preview` failures are pre-existing.

**Pull request closed without merging**